crates/matrix-sdk/src/sliding_sync.rs

```diff
@@ -275,6 +278,26 @@ impl SlidingSyncRoom {
         self.inner.name.as_deref()
     }
 
+    /// Is this a direct message?
+    pub fn is_dm(&self) -> Option<bool> {
+        self.inner.is_dm
+    }
+
+    /// Was this an initial response.
+    pub fn is_an_initial_response(&self) -> Option<bool> {
```

Comment:
```suggestion
    pub fn is_initial_response(&self) -> Option<bool> {
```

or just `is_initial` like in the FFI crate?

Reply:
I don't believe it conveys a good idea of what the method does :-/. “Is it an initial sliding sync room?” would be the first understanding of this, don't you think?

Reply:
Ah sorry, missed it. Yeah, perfect :-)!